ui/templates/welcome.html

```diff
@@ -6,9 +6,26 @@
 
 <p>
```

Comment:
That's correct: 

> My understanding of the comments above is that the "Welcome" page will contain a single link per repo, which brings you to the (possibly blank) listing page, and the listing page will now have the import link. Is that correct?

Sorry about the confusion. I think someone made a comment about going to an empty listing page, but the solution is to add an import link to the listing page, not on this welcome page. We want to direct the user flow always through the repository.